source/client-side-encryption/tests/generate.py

```diff
@@ -25,11 +25,11 @@
 filepath = sys.argv[1]
 filedir = os.path.dirname(filepath)
 (filename, ext) = os.path.splitext(os.path.basename(filepath))
-if ext != ".yml":
-    print("Input file must end with .yml")
+if ext != ".template":
+    print("Input file must end with .template.yml")
     sys.exit(1)
 (filename, ext) = os.path.splitext(filename)
-if ext != ".template":
+if ext != ".yml":
```

Comment:
I'm not sure how this check differs from the previous check, but is this still accurate?

Is this attempting to validate the output filename (derived from the original template filename)? Perhaps it's just the error message below that's inaccurate.

Reply:
Yeah the error message needed to be updated.

Reply:
Ah, I see what the logic is now. You're checking the order of extensions. :+1: